Core/Object Arts/Dolphin/Base/Package.cls

```diff
@@ -42,6 +42,15 @@ aboutBlockBytes: aByteArray
 
 !
 
+addAllPrerequisitesTo: aSet
+	"Private - Add to <aSet> all those Packages which must be loaded in
+	before the receiver may be loaded in."
+
+	self prerequisites
+		do: [:each | (aSet includes: each) ifFalse:
+					[aSet add: each.
```

Comment:
Right, I'm not used to that yet (since it wasn't there in D6). Thanks for the reminder.